src/App.tsx

```diff
@@ -1,61 +1,146 @@
+import { useState } from 'react';
+import { Todo } from './Types/Todo';
+import { User } from './Types/User';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+const getUserById = (todo: Todo) => {
+  return usersFromServer.find(user => user.id === todo.userId) || null;
+};
+
+const getTodos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUserById(todo),
+}));
+
+const createTodo = (
+  selectedUser: User,
+  title: string,
+  todos: Todo[],
+): Todo => {
+  const lastId = Math.max(...todos.map(todo => todo.id));
+
+  return ({
+    id: lastId + 1,
+    title,
+    completed: false,
+    userId: selectedUser.id,
+    user: selectedUser,
+  });
+};
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState(getTodos);
+  const [isUserError, setIsUserError] = useState(false);
+  const [titleError, setTitleError] = useState(false);
```

Comment:
if you have a boolean type - start name with `has`, `is`, something like that
```suggestion
  const [isTitleError, setIsTitleError] = useState(false);
```